tests/integration/events/test_rabbitmq_connector.py

```diff
@@ -0,0 +1,88 @@
+from unittest import mock
+from unittest.mock import PropertyMock
+
+import pytest
+from pika import BlockingConnection
+
+from petisco import RabbitMQConnector
+
+from petisco.events.rabbitmq.rabbitmq_is_running_locally import (
+    rabbitmq_is_running_locally,
+)
+
+
+@pytest.mark.integration
```

Comment:
@julian-lamoso @fgsalomon Can you think of any more tests?

Reply:
Nope. They're very comprehensive.